app/scripts/modules/core/src/managed/ManagedResourceHistoryModal.tsx

```diff
@@ -146,7 +146,7 @@ const renderExpandedRowContent = (
   dismissModal: () => any,
 ) => {
   return (
-    <div className="flex-container-v left">
+    <div className="flex-container-v left" style={{ maxWidth: '90vw' }}>
```

Comment:
nothing wrong with this per se, but you could also use the [built in max width controls](https://github.com/spinnaker/deck/pull/8237) on `Modal` if you like. I don't have a strong pref, the `Modal` API is a little more first class but this approach lives close to the content which is nice.